msal/src/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -59,6 +59,14 @@
     <None Include="**\*.cs;**\*.xml;**\*.axml" Exclude="obj\**\*.*;bin\**\*.*" />
     <Compile Remove="Platforms\**\*.*" />
     <Compile Remove="Features\**\*.*" />
+    <None Remove="Telemetry.cs" />
```

Comment:
You probably do not need these at all.

Reply:
Auto generated by visual studio. I can remove